microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/core/ls/JavaProjectDelegateCommandHandler.java

```diff
@@ -31,9 +37,29 @@ public JavaProjectDelegateCommandHandler() {
 	public Object executeCommand(String commandId, List<Object> arguments, IProgressMonitor progress) throws Exception {
 		switch (commandId) {
 		case PROJECT_LABELS_COMMAND_ID:
```

Comment:
should we deprecate this command?

Reply:
This command is used for this issue. Before this pr this command returned a List of project used by typescript. I renamed this command by adding a s. In other words the two commands are requiered

Reply:
So you changed the semantic of the existing command (bad) and the 2 command names are too close (bad).

Reply:
> So you changed the semantic of the existing command (bad) 

Indeed but it's only TypeScript which consumes it and the command name uses "`project`" although it's for `all project` and not for a given `project`

> and the 2 command names are too close (bad).

What is your suggestion?